parts/kuberneteskubelet.service

```diff
@@ -38,15 +38,13 @@ ExecStart=/usr/bin/docker run \
         --address=0.0.0.0 \
         --allow-privileged=true \
         --enable-server \
-        --enable-debugging-handlers \
```

Comment:
verified that docs say this option in enabled in `1.5`, `1.6`, and `1.7`

Reply:
Yes, though it's on by default already. That's why I removed it.

Reply:
Yes! That was my way of 👍 this line removal!